src/main/java/it/rate/webapp/controllers/api/InterestRestController.java

```diff
@@ -46,4 +48,10 @@ public ResponseEntity<?> getApplicantsByInterestId(@PathVariable Long interestId
     Interest interest = interestService.getById(interestId);
     return ResponseEntity.ok(interestService.getUsersDTO(interest, Role.RoleType.APPLICANT));
   }
+
+  @GetMapping("/{interestId}/places")
+  public ResponseEntity<?> getAllPlaceInfoDTOs(@PathVariable Long interestId) {
+    Interest interest = interestService.getById(interestId);

```

Comment:
One more thing - since this endpoint is not preauthorized there should probably be some check if interest exists before getting it by id.

Reply:
That doesn't seem necessary since there is no reason to access this method without accessing the interest first anyway.

Reply:
Yes but it is still an open Rest endpoint and should be functional on its own...